src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1791,12 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	private static final String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
+
 	public static void info(final CommandSender sender, final String info) {
-		sender.sendMessage(SKRIPT_PREFIX + Utils.replaceEnglishChatStyles(info));
+		sender.sendMessage(Utils.replaceEnglishChatStyles((SKRIPT_PREFIX_MESSAGE.toString().isEmpty() ? SKRIPT_PREFIX : SKRIPT_PREFIX_MESSAGE) + info));
```

Comment:
could this be cached instead of computing it every time we broadcast?

Reply:
I get what you mean by it but can you give a bit more information towards the cache part of it

Reply:
The Message class already effectively caches it so it should be okay as is

The actual value will only be recalculated when the language entries update

Reply:
I meant this part `(SKRIPT_PREFIX_MESSAGE.toString().isEmpty() ? SKRIPT_PREFIX : SKRIPT_PREFIX_MESSAGE)`, or if updating is an issue, it should be a private method to retrieve the prefix instead of pasting the same line over and over again

Reply:
Is it better do you think with the getValueOrDefault method?

Reply:
There is a [comment](https://github.com/SkriptLang/Skript/pull/5437#discussion_r1104244992) related

Reply:
> Is it better do you think with the getValueOrDefault method?

That would be better, but I think we could make a method called `getSkriptPrefix()` or something that returns a string *using* the Message, so it updates when lang is reloaded